### PR TITLE
Move DrawOutline into BaseWrapper

### PR DESCRIPTION
**Pull request by moden-py**

### CODE REVIEW

File: pywinauto/unittests/test_HwndWrapper.py [hunk 1]
```diff
@@ -319,6 +319,25 @@ def testGetProperties(self):
 #    def testCaptureAsImage(self):
 #        pass
```
Comment (moden-py) [inline pywinauto/unittests/test_HwndWrapper.py]: Disabling new tests since they would not work on AppVeyor environment. "Black screenshots" problem.

**Comment by airelil** [comment]
A small notice, a call ```self.assertEquals``` produces a deprecation warning on py3. Use ```self.assertEqual``` instead.

File: pywinauto/base_wrapper.py [hunk 2]
```diff
@@ -442,6 +443,70 @@ def get_properties(self):
...
+        fill=win32defines.BS_NULL,
+        rect=None):
+        """Draw an outline around the window.
+
+        * **colour** can be either an integer or one of 'red', 'green', 'blue'
+          (default 'green')
+        * **thickness** thickness of rectangle (default 2)
+        * **fill** how to fill in the rectangle (default BS_NULL)
+        * **rect** the coordinates of the rectangle to draw (defaults to
+          the rectangle of the control.
```
Comment (vasily-v-ryabov) [inline pywinauto/base_wrapper.py]: (minor) right bracket is missing after `control`.

File: pywinauto/base_wrapper.py [hunk 3]
```diff
@@ -442,6 +443,70 @@ def get_properties(self):
     GetProperties = get_properties
 
     #-----------------------------------------------------------
+    def draw_outline(
+        self,
+        colour='green',
```
Comment (vasily-v-ryabov) [inline pywinauto/base_wrapper.py]: `=` should be wrapped by space according to PEP-8.

File: pywinauto/base_wrapper.py [hunk 4]
```diff
@@ -442,6 +443,70 @@ def get_properties(self):
...
+        thickness=2,
+        fill=win32defines.BS_NULL,
+        rect=None):
+        """Draw an outline around the window.
```
Comment (vasily-v-ryabov) [inline pywinauto/base_wrapper.py]: should be
```
"""
Draw...
```

File: pywinauto/base_wrapper.py (hunk 3 above)
Comment (moden-py) [inline pywinauto/base_wrapper.py]: Vasily, this are the foo arguments. I am sure this not requires spaces around.
...
https://www.python.org/dev/peps/pep-0008/
"Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. "

Comment (vasily-v-ryabov) [inline pywinauto/base_wrapper.py]: Okay, I mixed it up with another case.

**Pull request merged**